docs/cli/etcd-snapshot.md

```diff
@@ -235,7 +187,7 @@ K3s runs through several steps when restoring a snapshot:
 Select the tab below that matches your cluster configuration.
 
 <Tabs queryString="etcdsnap">
-<TabItem value="Single Server">
+<TabItem value="Single Server" default>
```

Comment:
Nit: a default tab is not required if its the first tab defined.

Reply:
I did not know this, thanks